relayer/chains/substrate/event_parser.go

```diff
@@ -155,6 +161,7 @@ func genAccumKey(data interface{}) ibcPacketKey {
 
 // client info attributes and methods
 type clientInfo struct {
+	// TODO: is there a reason the fields are exported? Since the clientInfo struct is not exported.
```

Comment:
In Go it is technically valid to have an unexported type with exported fields. You wouldn't be able to initialize a variable of type `clientInfo` from another package but you could have an exported function that returns type `clientInfo` and then the exported fields are accessible. 

If the fields are never being accessed outside of this package though I think it makes sense to make them unexported. 

See: https://www.ardanlabs.com/blog/2014/03/exportedunexported-identifiers-in-go.html